common/json.hpp

```diff
@@ -102,6 +102,9 @@ class Json {
 	ExpectedDouble GetDouble() const;
 	ExpectedBool GetBool() const;
 
+	template <typename T>
+	expected::expected<T, error::Error> Get() const;
```

Comment:
We should add at least some trivial unit test. If nothing else than to just avoid regressions in the future.

Reply:
👍